server/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
```

Comment:
in general implement a --help flag.
So that user can see the usage on doing `./server --help`

Reply:
that is automatically generated and I can see the instructions by running `./server --help`